cspell-config/cspell-circom.txt

```diff
@@ -0,0 +1,7 @@
+zkey
```

Comment:
You'll want to add this file under the dictionaryDefinitions in the cspell.json file and then list it in the dictionaries to get it added for spell checks.